README.md

```diff
@@ -170,6 +170,8 @@ PostSchema.plugin(patchHistory, {
   An array of two functions that generate model and collection name based on the `name` option. Default: An array of [humps](https://github.com/domchristie/humps).pascalize and [humps](https://github.com/domchristie/humps).decamelize
 * `includes` <br/>
   Property definitions that will be included in the patch schema. Read more about includes in the next chapter of the documentation. Default: `{}`
+* `excludes` <br/>
+  Property paths that will be excluded in patches. Read more about excludes in the chapter after next. Default: `[]`
```

Comment:
```md
Read more about excludes in the [excludes chapter of the documentation](#excludes)
```

Reply:
Awesome. You can link the chapter in markdown - in case the documentation grows and something goes in-between, people can jump to it directly.